spec/support/locker_is_included_shared_examples.rb

```diff
@@ -128,7 +128,7 @@
     it 'is chainable' do
       expect do
         criteria = model.where(_id: 1).unlocked.where(_id: 2)
-        expect(criteria.selector['_id']).to eq(2)
+        expect(criteria.selector['_id']).to eq(Mongoid::VERSION < '7' ? 2 : 1)
```

Comment:
You can't compare versions like that. It will break at Mongoid 10 for example.

Use https://github.com/dblock/mongoid-compatibility or roll your own, see https://code.dblock.org/2020/07/16/comparing-version-numbers-in-ruby.html

Reply:
Done, updated code to use `mongoid-compatibility`